ticdc/troubleshoot-ticdc.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Handle TiCDC Issues
+summary: Learn how to handle TiCDC issues.
+category: reference
+---
+
+# Handle TiCDC Issues
+
+This document introduces the common issues and errors that you might encounter when using TiCDC, and the corresponding maintenance and troubleshooting methods.
+
+## How to choose `start-ts` when starting a task
+
+The `start-ts` of a replication task corresponds to a Timestamp Oracle (TSO) in the upstream TiDB cluster. TiCDC requests data from this TSO in a replication task. Therefore, the `start-ts` of the replication task must meet the following requirements:
+
+- The value of `start-ts` is larger than the `tikv_gc_safe_point` value of the current TiDB cluster. Otherwise, you will fail to create a task.
```

Comment:
```suggestion
- The value of `start-ts` is larger than the `tikv_gc_safe_point` value of the current TiDB cluster. Otherwise, an error occurs when you create a task.
```